local_php.ini

```diff
@@ -7,6 +7,15 @@
 ; Disable aggressive caching
 opcache.enable = 0
 
+; Enable xdebug
+; https://enrise.com/2018/02/debugging-php-with-xdebug/
+zend_extension=/app/.heroku/php/lib/php/extensions/no-debug-non-zts-20131226/xdebug.so
```

Comment:
I think this path is standard across all docker instances since it uses the published date for xdebug-2.4